corehq/apps/users/migrations/0059_add_web_apps_groups_permission_flag.py

```diff
@@ -0,0 +1,22 @@
+from django.db import migrations
+
+from corehq.apps.cloudcare.models import ApplicationAccess
+from corehq.toggles import NAMESPACE_DOMAIN, WEB_APPS_PERMISSIONS_VIA_GROUPS
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def enable_flag(apps, schema_editor):
+    for access in ApplicationAccess.objects.filter(restrict=True):
```

Comment:
Is this as targeted as we can safely do?  Like, would it make sense to check whether the domain has access to web apps or groups configured or some such?  I guess that could also be a clean-up pass done separately later

Reply:
I think it's pretty harmless, the view itself is still decorated to require the cloudcare privilege. The risks here are:
* Block access to this page for a domain that should have it: I think this is unlikely, although I could remove the `restrict=True` filter.
* Accidentally give access to a domain that doesn't need it: not a big deal, since right now all domains (with cloudcare privilege) can access this page

Reply:
> Accidentally give access to a domain that doesn't need it: not a big deal, since right now all domains (with cloudcare privilege) can access this page
Yeah, I agree - this is the situation I was thinking of.  Fewer domains on this flag means that it's easier to eventually remove in the future, but I agree it's probably better to play it safe here and work on whittling down the list separately whenever we get around to it